kcidb/test_orm.py

```diff
@@ -498,3 +498,494 @@ def test_repr(self):
         self.assertEqual(repr(pattern(None, True, "checkout",
                               {("",)})),
                          ">checkout[\"\"]#")
+
+
+def raw_checkout(**kwargs):
+    """
+    Template for checkout type
+    Args:
+        kwargs:  A dictionary containing non-null fields.
+
+    Returns:
+        The dictionary for checkout type
+         with the missing fields set to None.
+    """
+    raw_data = {}
+    for item in kcidb.orm.SCHEMA.types["checkout"] \
+            .json_schema['properties'].keys():
+        raw_data[item] = kwargs.get(item, None)
+    return raw_data
```

Comment:
Instead of copying this code between the three or four (don't you need `raw_revision` as well?), could you please extract the common part into a function called `raw_data` and use it in all of these? It would accept the type name as a positional argument and the `**kwargs` as the field dictionary.